### PR TITLE
🩹 Temp fix for balance issue

### PR DESCRIPTION
**Pull request by vikiival**
- :adhesive_bandage: balance
- :zap: use getter instead of direct value

## TL;DR

The problem is that nested objects in Pinia are not reactive, so you need to set them beforehand to work smoothly.
The problem is that is really wordy.

I tried to use `Vue.set` as mentioned in https://github.com/vuejs/pinia/issues/43#issuecomment-1373622966 but without any success.

Feel free to fix better

**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 \_\_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] ref #5838

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [x] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [x] I've found edge cases

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

Screenshot 2023-05-17 at 13 07 37

### <samp>🤖 Generated by Copilot at f3eed89</samp>

Refactored the `identity.ts` store and the `AccountBalance.vue` component to improve type safety and reactivity of the balance data. Used type aliases, constants, and a getter method to access and manipulate the balance and token maps of the `Auth` interface.

### <samp>🤖 Generated by Copilot at f3eed89</samp>

> _To keep the balance reactive and true_
> _We use a getter method from the store_
> _We also refactor the types_
> _And use some constants nice_
> _For the identity data we adore_

### CODE REVIEW

File: stores/identity.ts [hunk 1]
```diff
@@ -4,12 +4,25 @@ import { defineStore } from 'pinia'
...
+  glmr: '0',
+  movr: '0',
+}
 
 export interface IdentityMap {
   [address: string]: Registration
 }
 
-type BalanceMap = Record<string, string>
+type NumericId = `${number}`
```
Comment (roiLeo) [inline stores/identity.ts:23]: wut?
```suggestion
type NumericId = number
```

File: components/shared/AccountBalance.vue [hunk 2]
```diff
@@ -14,5 +14,5 @@ const Money = defineAsyncComponent(
 const { urlPrefix } = usePrefix()
```
Comment (roiLeo) [inline components/shared/AccountBalance.vue:14]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Comment by yangwao** [comment]
yes!

image

**Pull request merged**